Core.ApplicationServices/UserService.cs

```diff
@@ -306,6 +307,15 @@ public void Dispose()
                 .Transform(Result<IQueryable<User>, OperationError>.Success);
         }
 
+        public Result<UserDeletionStrategyType, OperationError> GetUserDeletionStrategy(int userId)
+        {
+            var user = _userRepository.AsQueryable().ById(userId);
```

Comment:
you are missing the org id reference to scope the "question"...
Also, no everyone can call this method, so if you are not global or local admin (for the org in question) the result will be forbidden

Logic should be:
```
  if User is deleted
      ERROR(badinput)
  else if User has no roles in the organization
      ERROR(badinput)
  else if Authenticated user is Local admin or Global Admin
      If User is Global Admin
          return "Local"
      else if (User has roles in other orgs than the one in question)
          return "Local"
     else
          return "Global"
   else
         return ERROR(forbidden)
```